.github/workflows/size-limit.yml

```diff
@@ -0,0 +1,16 @@
+name: 'size'
+
+on: pull_request
+  branches:
+    - main
```

Comment:
Heads up there is an error on this part, the action is not reporting back to the PR
https://github.com/contentful/forma-36/actions/runs/2962500861/workflow

Reply:
@denkristoffer it's still failing to build 🤔

Reply:
@mshaaban0 It's not going to work for this PR, but you can see it working here: https://github.com/contentful/forma-36/runs/8136279821?check_suite_focus=true

The reason it fails here is that the action works by running `build` and then `npx size-limit --json` first on the branch of the PR, then on `main`, and it compares the two size results. Before this PR was merged the configuration for `size-limit` was broken on the `main` branch, so it couldn't compare the results and then exited with an error. It should work for future PRs, although it looks like the config needs to be tweaked.